### PR TITLE
fix: ai-srv OCI image build

### PR DESCRIPTION
**Pull request by stb1337**
See #15 

@kay-schmitteckert

### CODE REVIEW

File: multi-tenant/code/package.json [hunk 1]
```diff
@@ -101,6 +101,7 @@
         "--------------------": "",
         "--------BUILD-------": "",
         "/------------------/": "",
+        "start": "cds-serve",
```
Comment (kay-schmitteckert) [inline multi-tenant/code/package.json:104]: shouldnt be part of the build section, rather top

File: multi-tenant/deploy/kyma/package.json [hunk 2]
```diff
@@ -5,6 +5,7 @@
   "license": "Apache License 2.0",
   "description": "AI SaaS",
   "scripts": {
+    "start": "cds-serve",
```
Comment (kay-schmitteckert) [inline multi-tenant/deploy/kyma/package.json:8]: not needed in the package.json for deployment

File: single-tenant/code/package.json [hunk 3]
```diff
@@ -66,6 +66,7 @@
         "cap": "cds-ts watch --profile hybrid",
         "router": "npm run start:local --prefix ./router",
         "--------UI DEPLOYMENT-------": "",
+        "start": "cds-serve",
```
Comment (kay-schmitteckert) [inline single-tenant/code/package.json:69]: move to top

File: single-tenant/deploy/kyma/package.json [hunk 4]
```diff
@@ -30,7 +31,7 @@
     "--- RUN LIKE npx --yes cross-env IMAGE_PREFIX=sap-demo npm run build:all ---": "",
     "/------------------------/": "",
     "build:all": "npm run build:srv && npm run build:db && npm run build:router && npm run build:html5-deployer",
-    "build:srv": "cross-env-shell pack build $IMAGE_PREFIX/ai-srv -p ../../code/gen/srv -B paketobuildpacks/builder:base -b paketo-buildpacks/nodejs -e BP_LAUNCHPOINT=./node_modules/@sap/cds/bin/cds-serve.js -e BP_NODE_RUN_SCRIPTS=",
+    "build:srv": "cross-env-shell pack build $IMAGE_PREFIX/ai-srv -p ../../code/gen/srv --buildpack gcr.io/paketo-buildpacks/nodejs --builder paketobuildpacks/builder-jammy-base -e BP_LAUNCHPOINT=./node_modules/@sap/cds/bin/cds-serve.js",
```
Comment (kay-schmitteckert) [inline single-tenant/deploy/kyma/package.json:34]: i just wonder if we still need BP_LAUNCHPOINT if we define the start script in `code/package.json`

**Review by kay-schmitteckert — CHANGES_REQUESTED** [review]
Thanks for your contribution. Are you willing to take a look at the comments and adjust according to it? Otherwise it's also fine that I merge the state and incorporate after.

File: single-tenant/deploy/kyma/package.json (hunk 4 above)
Comment (stb1337) [reply single-tenant/deploy/kyma/package.json:33]: i am not sure. Lets keep it that way for the first iteration.

**Review by kay-schmitteckert — APPROVED** [review]

**Pull request merged**